### PR TITLE
Use full DHT bootstrap list and update deps

### PR DESCRIPTION
**Pull request by anacrolix**
Uses the full DHT bootstrap list. Builds on https://github.com/libp2p/go-libp2p-examples/pull/40.

### CODE REVIEW

**Comment by anacrolix** [comment]
The CircleCI should pass when #46 is merged.

File: go.mod [hunk 1]
```diff
@@ -33,7 +40,7 @@ require (
 	github.com/libp2p/go-libp2p-host v3.0.15+incompatible
 	github.com/libp2p/go-libp2p-interface-connmgr v0.0.21 // indirect
 	github.com/libp2p/go-libp2p-interface-pnet v3.0.0+incompatible // indirect
-	github.com/libp2p/go-libp2p-kad-dht v4.4.12+incompatible
+	github.com/libp2p/go-libp2p-kad-dht v4.4.13-0.20190129000003-bebd753a1d86+incompatible
```
Comment (raulk) [inline go.mod]: This hard dependency on a specific commit is not good. Can't we refer to master HEAD?

**Comment by anacrolix** [comment]
It's also worth pointing out that it won't fall out of sync. Downstreams or upstreams (visibly) will raise the minimum version. There's no lockstep nonsense.

File: go.mod (hunk 1 above)
Comment (anacrolix) [reply go.mod]: This is addressed now, but I think the original concern is ill-conceived.

**Comment by anacrolix** [comment]
I've bumped the deps after observing https://github.com/libp2p/go-libp2p/issues/529#issuecomment-461175029.

**Comment by anacrolix** [comment]
@raulk let's get this merged.

**Pull request merged**